underscore.js

```diff
@@ -1328,7 +1333,10 @@
   // Shortcut function for checking if an object has a given property directly
   // on itself (in other words, not on a prototype).
   _.has = function(obj, key) {
-    return obj != null && hasOwnProperty.call(obj, key);
+    if (!_.isObject(obj)) return false;
```

Comment:
`_.isObject(obj)` is not exactly equivalent to `obj != null`, so this is a subtle but potentially very nasty breaking change.